docs-crawler/docs/aggregate_crawler_output.py

```diff
@@ -1,6 +1,18 @@
 import json
 import re
 
+TRANSLATIONS = {
```

Comment:
So these are the translations that I moved into our crawler which will do word level translations for arguments & function names.